lib/commands/kendoui-notes.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUINotesCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	canExecute(args: string[]): IFuture<boolean> {
+		return ((): boolean => {
+			let number = args ? parseInt(args[0]) : true;
+			if (!number) {
+				this.$errors.failWithoutHelp(`The argument ${args[0]} is not applicable to this command`);
+			}
+
+			if(args.length > 1) {
+				this.$errors.fail("This command only accepts one parameter.");
+			}
+
+			if(this.$options.core && this.$options.professional) {
+				this.$errors.fail("You cannot specify core and professional options simultaneously.");
```

Comment:
This code should actually not be here, my mistake - the whole `canExecute` method is redundant here